libvmaf/src/feature/cambi.c

```diff
@@ -147,6 +172,13 @@ static const VmafOption options[] = {
         .min = 0,
         .max = 5,
     },
+    {
+        .name = "full_reference",
```

Comment:
Should we abbreviate as `full_ref`?

Reply:
agreed, `full_reference` is a bit unwieldy